.travis.yml

```diff
@@ -149,6 +148,7 @@ before_script:
 
   # show some versions and env information
   - php --version
+  - php --ini
```

Comment:
looks like this breaks HHVM tests, it starts the binary which is waiting for input.